docs/installation.md

```diff
@@ -218,19 +218,174 @@ On Linux, as an optional post-installation task, you may wish to setup the bot t
+conda config --set auto_activate_base false
+```
+
+Conda as a package manager can download new packages from "channels". The best developed Conda channel, is not the default channel and is called `conda-forge`. The code below switches to it.
+
+```bash    
+# adding forge
+conda config --env --add channels conda-forge
 
-``` bash
-conda env create -f environment.yml
+# make it strict
+conda config --env --set channel_priority strict 	
 ```
 
+You can check the status of conda with the following code.
+
+```bash
+conda info
+conda config --show channels
+conda config --show channel_priority
+```
+
+#### 3. Freqtrade Download
+
+Download and install freqtrade.
+
+```bash
+# download freqtrade
+git clone https://github.com/freqtrade/freqtrade.git
+
+# enter downloaded directory 'freqtrade'
+cd freqtrade      
+```
+
+#### 4. Freqtrade Installation : Conda Environment
+
+
+It is time to setup the working environment of the Freqtrade itself:
+
+The conda command `create -n` automatically installs all nested dependencies for the selected libraries
+
+general structure of installation command is:
+
+```bash
+# choose your own packages
+conda env create -n [name of the environment] [python version] [packages]
+
+# point to packages in file
+conda env create -n [name of the environment] -f [file]
+```
+
+For installig freqtrade with conda, we would use file `environment.yml` which exist in freqtrade directory
+
+```bash
+conda env create -n freqtrade-conda -f environment.yml
+```
+
+#### 4. Install python dependencies
+
+```bash
+python3 -m pip install --upgrade pip
+python3 -m pip install -e .
+```
+
+Further read on the topic:
+
+https://towardsdatascience.com/a-guide-to-conda-environments-bc6180fc533?gi=1db972389cd1
+
+https://metager.de/meta/meta.ger3?eingabe=ardsdatascience+guide+to+conda+environment
+
+!!! Note "New heavy packages"
+    It may happen that creating a new Conda environment, populated with selected packages at the moment of creation, takes less time than installing a large, heavy dependent, GUI package, into previously set environment. Great example: Spyder
+
+
+##### pip install within conda, a reminder:
+
+The documentation of conda says that pip should NOT be used within conda, because internal problems can occur.
+However, they are rare. https://www.anaconda.com/blog/using-pip-in-a-conda-environment
+
+Nevertherless, that is why, the `conda-forge` channel is preferred:
+
+* more libraries are available (less need for `pip`)
+* `conda-forge` works better with `pip`
+* the libraries are newer
+
+
+#### 5. You are ready
+
+You are ready to run, create the user directory and configuration file [Bot Configuration](configuration.md), run the program `dry_run: True` to verify that everything is working, and run a backtest to double check.
+
+```bash
+# Prerequisite
+freqtrade create-userdir --userdir user_data
+
+# set up config file
+# make sure to run it set the `dry_run : true` as you start
+freqtrade new-config --config config.json
```

Comment:
this MUST point out to enter (activate) the environment first. Otherwise people will fail. 
It's clear to us - but not to someone unfamiliar with conda / virtual environments in general.

Reply:
Yes, it is a typ-o